### ISSUE TITLE
All in one?

### ISSUE THREAD

**Issue by ainslec** [DominoKit/domino-ui#79]
Hello,

I'm wondering if it's possible to build Domino so that it doesn't require to access any CSS or HTML  accessed externally? That is, everything is baked into the compiled JavaScript.

Could this be achieved, or will it never happen?

Chris.

**Comment by vegegoku** [comment]
The css that domino-ui uses is part of the domino-ui jar, it does not reference any css or html from external source.
or do you mean to auto inject the css in the page?

**Comment by ainslec** [comment]
Yes, I mean auto-injection, have you considered it? Or maybe make auto-injection configurable?

**Comment by vegegoku** [comment]
this is already in the road map, once we reach the styles restructuring we will implement it.
